### DISCUSSION TITLE
A collection of default themes

### DISCUSSION THREAD

**Discussion by lazarusA** [squidfunk/mkdocs-material#4914] [Ideas]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I want to suggest an idea and checked that ...

- [ ] ... to my best knowledge, my idea wouldn't break something for other users
- [ ] ... the documentation does not mention anything about my idea
- [X] ... there are no open or closed issues that are related to my idea

### Description

Some months ago I saw a tweet asking about the need for more theme styles for material docs. However, I don't know of find any issue tracking this feature or even if there are plans for it.

### Use Cases

It would be nice to have/create some webpages that don't look all the same.

**Comment by squidfunk** [comment]
This is actually an excellent opportunity for third-party extensions or CSS – it's trivial to add via customization. I don't think I will have the time to work on other layouts anytime soon, but as said, it's easy to change styles via theme extension 😊

I'm converting this to a discussion, so you and others can share and talk about some ideas.
